### PR TITLE
Intro-Python-I | Alex King

### PR DESCRIPTION
**Pull request by Alex-AK**

### CODE REVIEW

File: src/printing.py [hunk 1]
```diff
@@ -12,6 +12,11 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print("x is %d, y is %f, z is %s" % (x, round(y, 2), z))
+
 # Use the 'format' string method to print the same thing
+print("x is {}, y is {}, z is {}".format(x, round(y, 2), z))
```
Comment (koennecker) [inline src/printing.py]: This works, but there's also a way to do the rounding inside the formatted string itself. Check out this link for the first two problems https://mkaz.blog/code/python-string-format-cookbook/

File: src/printing.py [hunk 2]
```diff
@@ -12,6 +12,11 @@
...
+
+# Finally, print the same thing using an f-string
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print(F"x is {x}, y is {round(y, 2)}, z is {z}")
```
Comment (koennecker) [inline src/printing.py]: Same as above, check out this link for how to round with f-strings https://stackoverflow.com/questions/45310254/fixed-digits-after-decimal-with-f-strings

File: src/printing.py (hunk 1 above)
Comment (Alex-AK) [reply src/printing.py]: Awesome, I'll change it. That syntax is much nicer.

File: src/comprehensions.py [hunk 3]
```diff
@@ -12,14 +12,23 @@
 
 y = []
 
-print (y)
+for x in range(5):
```
Comment (koennecker) [inline src/comprehensions.py:15]: The problems on line 15, 28, and 40 are asking you to use a list comprehension. Can you think of a way to make them like the one on line 52?

**Comment by koennecker** [comment]
Great start on the first day of the project, Alex!

File: src/args.py [hunk 4]
```diff
@@ -6,29 +6,55 @@
...
 
+
+# def f2(*numbers):
+#     total = 0
+#     for number in numbers:
+#         total = total + number
+#     return total
+
+def f2(*numbers):
+    return sum(numbers)
```
Comment (koennecker) [inline src/args.py:29]: Nice usage of `sum`.

**Comment by koennecker** [comment]
Everything looks good. Great job with the frequent and descriptive commits.

**Pull request merged**